src/dist/worker.h

```diff
@@ -72,4 +85,5 @@ class Worker {
   std::string response_queue_address;
```

Comment:
these address strings are only referenced in one place and probably can just be local vars and not private members?